rust/datafusion/src/execution/context.rs

```diff
@@ -119,7 +120,7 @@ impl ExecutionContext {
         }
     }
 
-    /// of RecordBatch instances)
+    /// Creates a dataframe that will execute a SQL query.
```

Comment:
Please review. My understanding is that this does not execute the query immediately, but creates a plan for it.

The original comment was from v1.0.1: `Execute a SQL query and produce a Relation (a schema-aware iterator over a series of RecordBatch instances)`. By v2.0.0 the methods was returning a `DataFrame` (not `RecordBatch`es), and by v3.0.0 the comment was mangled.

Reply:
Good point. Yes, this solely creates `DataFrame` corresponding to the passed SQL.